dev-util/rpi-imager/rpi-imager-1.8.5.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Raspberry Pi Imaging Utility"
+HOMEPAGE="
+https://www.raspberrypi.com/software/
+https://www.raspberrypi.com/documentation/computers/getting-started.html#raspberry-pi-imager
+https://github.com/raspberrypi/rpi-imager
+"
+SRC_URI="https://github.com/raspberrypi/rpi-imager/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}/src"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+RDEPEND="
```

Comment:
I would start with something like this. It's not perfect and likely missing something. But as I said, putting everything that comes out of ldd into RDEPND is a bit rough.
```
RDEPEND="
	app-arch/libarchive:=
	app-arch/xz-utils
	net-misc/curl
	sys-apps/util-linux
	sys-libs/zlib
	gnutls? ( net-libs/gnutls:= )
	!gnutls? ( dev-libs/openssl:= )
	qt6? (
		dev-qt/qtbase:6[concurrent,dbus,gui,network,widgets]
		dev-qt/qtdeclarative:6[svg]
	)
	!qt6? (
		dev-qt/qtconcurrent:5
		dev-qt/qtcore:5
		dev-qt/qtdbus:5
		dev-qt/qtdeclarative:5
		dev-qt/qtgui:5
		dev-qt/qtnetwork:5
		dev-qt/qtquickcontrols2:5
		dev-qt/qtsvg:5
		dev-qt/qtwidgets:5
	)
"
DEPEND="${RDEPEND}"
BDEPEND="
	qt6? ( dev-qt/qttools:6[linguist] )
	!qt6? ( dev-qt/linguist-tools:5 )
"
```

Reply:
Thanks for the help. I am looking into how you did these so far and will look into CMakeLists more.

Reply:
A quick glance at CMakeLists will give you a general idea of the dependencies. 
For example,
https://github.com/raspberrypi/rpi-imager/blob/4b6ee9bb4b1f29587db6145d82f4784d8863d036/src/CMakeLists.txt#L42-L50
need gnutls _or_ openssl

https://github.com/raspberrypi/rpi-imager/blob/4b6ee9bb4b1f29587db6145d82f4784d8863d036/src/CMakeLists.txt#L97-L104
need qt5 _or_ qt6

https://github.com/raspberrypi/rpi-imager/blob/4b6ee9bb4b1f29587db6145d82f4784d8863d036/src/CMakeLists.txt#L331
need curl

https://github.com/raspberrypi/rpi-imager/blob/4b6ee9bb4b1f29587db6145d82f4784d8863d036/src/CMakeLists.txt#L335
need lsblk (util-linux)